### PR TITLE
adding more info about PAT

### PR DESCRIPTION
**Pull request by svensht2**

### CODE REVIEW

**Comment by svensht2** [comment]
@jstys92 I dont know that you need to add anything, maybe just look at it and let me know if anything is missing
@willkelleher there are two sections:
**PAT with webplayer**

**PAT with 8thwall**

That could use a blurb that its supported and a link

File: docs/source/Components/index.rst [hunk 1]
```diff
@@ -56,7 +56,44 @@ PAT Components
...
 | There are example PAT scenes in 8i/examples/scenes that provide the best starting point.
+**What is PAT**
 
+
+PAT is the latest codec developed by 8i. PAT is a mesh based codec, in which the final
+asset output is represented as a sequence of textured meshes
+
+**Output from PAT**
+
+The PAT codec produces several outputs. Its important to understand that the outputs
```
Comment (jstys92) [inline docs/source/Components/index.rst]: It's

File: docs/source/Components/index.rst [hunk 2]
```diff
@@ -56,7 +56,44 @@ PAT Components
...
+
+**Difference from HVR**
+
+The main difference between HVR and PAT is how the underlying 3D asset is represented.
+HVR takes the approach of representing the 3D asset as a point cloud. However, PAT represents the asset as a textured mesh.
+Effectively, PAT decouples the mesh(geometry) resolution from the texture(color) resolution. Allowing us to scale up the asset perceived resolution, with smaller increase in overall asset size.
+To illustrate it, think of an actor with a very textured shirt, the underlying geometry of the shirt is fairly simple, allowing us to save a lot of geometry bandwidth. In this case most the complexity is on the texture side, which we can very effectively encode using standard video codecs.
+The represent the high details with HVR, we will need a very high resolution of points, causing the file size to explode.
+
+**Unity usage with Unity**
```
Comment (jstys92) [inline docs/source/Components/index.rst]: PAT usage

File: docs/source/Components/index.rst [hunk 3]
```diff
@@ -56,7 +56,44 @@ PAT Components
...
+
+HVR is using a point cloud to reprensent the data. Point clouds are less standatd and hence Unity doesnt do as good job rendering them.
```
Comment (jstys92) [inline docs/source/Components/index.rst]: *represent, standard, doesn't
*as good of a job

File: docs/source/Components/index.rst [hunk 4]
```diff
@@ -56,7 +56,44 @@ PAT Components
...
+
+    FBX - This output is primarily used for cases where the output needs to be edited using
+    tools like Maya. After editing, this format can be converted to any of the other formats.
+
+**Difference from HVR**
+
+The main difference between HVR and PAT is how the underlying 3D asset is represented.
+HVR takes the approach of representing the 3D asset as a point cloud. However, PAT represents the asset as a textured mesh.
+Effectively, PAT decouples the mesh(geometry) resolution from the texture(color) resolution. Allowing us to scale up the asset perceived resolution, with smaller increase in overall asset size.
+To illustrate it, think of an actor with a very textured shirt, the underlying geometry of the shirt is fairly simple, allowing us to save a lot of geometry bandwidth. In this case most the complexity is on the texture side, which we can very effectively encode using standard video codecs.
```
Comment (jstys92) [inline docs/source/Components/index.rst]: could end the first sentence at "very textured shirt"

File: docs/source/Components/index.rst [hunk 5]
```diff
@@ -56,7 +56,44 @@ PAT Components
 
 | The easiest way to instantiate a PAT playback object is via the example PatActor prefab.
 | There are example PAT scenes in 8i/examples/scenes that provide the best starting point.
+**What is PAT**
```
Comment (jstys92) [inline docs/source/Components/index.rst:56]: May need a line break before this

**Comment by jstys92** [comment]
In terms of "usage in Unity" were they looking more for a base example of how to set up PAT playback from scratch? We have an example prefab, but it's supposed to serve as an example, so it may be useful to have something else that's the base minimum steps to get playback to work (I could add that)

**Comment by svensht2** [comment]
I'm realizing I forgot to add file structure of the MPD. I'll work on figuring out how to add those graphics.
@jstys92 this example you're talking about, it is going to be code snippets or how are you thinking about it looking like

**Comment by jstys92** [comment]
Not so much code, more of a basic scene set up. "Drag this script onto object, link this object to that object" kind of thing. Similar to this section for HVR: https://8iunityplugin.readthedocs.io/en/latest/Quickstart.html#creating-a-hvractor

**Comment by svensht2** [comment]
yeah, I say it makes sense to match this HVR thing

**Comment by jstys92** [comment]
I pushed a commit which adds a section on PAT to the quickstart page

**Comment by svensht2** [comment]
@jstys92 addressed comments and added MPD structure description section

**Comment by svensht2** [comment]
here is the image for reference:
MPD structure

**Comment by svensht2** [comment]
forgot to add audio

File: docs/source/Components/index.rst [hunk 6]
```diff
@@ -64,36 +65,38 @@ asset output is represented as a sequence of textured meshes
...
+To illustrate it, think of an actor with a very textured shirt. The underlying geometry of the shirt is fairly simple, allowing us to save a lot of geometry bandwidth. In this case most the complexity is on the texture side, which we can very effectively encode using standard video codecs.
 The represent the high details with HVR, we will need a very high resolution of points, causing the file size to explode.
 
-**Unity usage with Unity**
+**PAT usage with Unity**
 
-HVR is using a point cloud to reprensent the data. Point clouds are less standatd and hence Unity doesnt do as good job rendering them.
+HVR is using a point cloud to represent the data. Point clouds are less standard and hence Unity's standard rendering will not produce the best output to the user.
 As a result, to produce the best quality render, the HVR plugin takes over the rendering of the asset. This internal rendering takes away some of feature set that a developer would expect with a 3D asset in Unity.
 In contrast, PAT simply delivers a mesh and a texture per frame, giving the developer back the control over rendering and 3D asset manipulation and effects.
```
Comment (jstys92) [inline docs/source/Components/index.rst]: Maybe "giving back the control over rendering and 3D asset manipulation and effects to the developer."

File: docs/source/Components/index.rst [hunk 7]
```diff
@@ -56,6 +56,7 @@ PAT Components
 
 | The easiest way to instantiate a PAT playback object is via the example PatActor prefab.
```
Comment (jstys92) [inline docs/source/Components/index.rst]: Should we maybe move this down to the more Unity centric section, and let the general overview come first?

File: docs/source/Components/index.rst [hunk 8]
```diff
@@ -64,36 +65,38 @@ asset output is represented as a sequence of textured meshes
...
-**Unity usage with Unity**
+**PAT usage with Unity**
 
-HVR is using a point cloud to reprensent the data. Point clouds are less standatd and hence Unity doesnt do as good job rendering them.
+HVR is using a point cloud to represent the data. Point clouds are less standard and hence Unity's standard rendering will not produce the best output to the user.
 As a result, to produce the best quality render, the HVR plugin takes over the rendering of the asset. This internal rendering takes away some of feature set that a developer would expect with a 3D asset in Unity.
 In contrast, PAT simply delivers a mesh and a texture per frame, giving the developer back the control over rendering and 3D asset manipulation and effects.
 
-**PAT with webplayer**
+**PAT with webplayer and 8thwall**
```
Comment (jstys92) [inline docs/source/Components/index.rst]: Maybe then have this section come at the very end as more of a footnote, so the Unity components follow more naturally from the section before

File: docs/source/Components/Mpd.rst [hunk 9]
```diff
@@ -0,0 +1,17 @@
...
+**Description of files and folders**
+
+
+* video - Folder that contains all texture data
+* mesh - Folder that contains all mesh data
+* manifest.mpd - MPEG-DASH manifest file describing the video/audio/mesh representation available for this asset
+* fallback.mp4 - video and audio mp4 that is used if the platform does not support MPEG-DASH or HLS
+* master_15.m3u8 - Manifest file for HLS streaming for 15 FPS variant of the asset
+* master_30.m3u8 - Manifest file for HLS streaming for the 30 FPS variant of the asset
+* init.mp4 - the init segment of the chunked video data
```
Comment (jstys92) [inline docs/source/Components/Mpd.rst]: could be more general with just "chunked data"

File: docs/source/Components/Mpd.rst [hunk 10]
```diff
@@ -0,0 +1,17 @@
...
+* *.m4s - chunks of the video data
```
Comment (jstys92) [inline docs/source/Components/Mpd.rst]: Need to \ escape the * in *.m4s so it doesn't get parsed into a link in the final doc

File: docs/source/Components/Mpd.rst [hunk 11]
```diff
@@ -0,0 +1,17 @@
+MPD structure
```
Comment (jstys92) [inline docs/source/Components/Mpd.rst:3]: Add :orphan: and a new line here at the top to squash a warning about this not being included in any octree

**Comment by jstys92** [comment]
Oh, the image has 0000.m4s for subsequent chunks - should increment

**Comment by svensht2** [comment]
all valid comments, addressed

**Review by jstys92 — APPROVED** [review]

**Pull request merged**